Resources/Prototypes/Entities/Markers/Spawners/Random/techboard.yml

```diff
@@ -27,18 +27,18 @@
     - WeaponCapacitorRechargerCircuitboard # Delta V - Add additional boards
     - StationMapCircuitboard # Delta V - Add additional boards
     - DeepFryerMachineCircuitboard # Delta V - Add additional boards
-    chance: 0.95
+    - ResearchAndDevelopmentServerMachineCircuitboard # Floof - Increase chance for R&D Server Boards
+    chance: 0.9
     rarePrototypes:
     - CommsComputerCircuitboard
-    #- ShuttleConsoleCircuitboard # Delta V - Remove shuttle boards
+    - ShuttleConsoleCircuitboard # Floof - Added Shuttle boards back
     - ComputerMassMediaCircuitboard
     - AutolatheMachineCircuitboard
     - ProtolatheMachineCircuitboard
     - SalvageMagnetMachineCircuitboard
     - PortableGeneratorSuperPacmanMachineCircuitboard # Delta V - Add additional boards
     - PortableGeneratorJrPacmanMachineCircuitboard # Delta V - Add additional boards
     - MedicalTechFabCircuitboard # Delta V - Add additional boards
-    - ResearchAndDevelopmentServerMachineCircuitboard # Delta V - Add additional boards
```

Comment:
Doesn't this remove the R&D server board from the pool?

Reply:
It gets moved from line 41 to line 30

Reply:
> It gets moved from line 41 to line 30

I am blind lol